docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -619,6 +619,154 @@ eclagent:
+      queues for each Thor definition and using a common name as an auxiliary
+      queue.</para>
+
+      <orderedlist>
+        <listitem>
+          <para>If you have not already added the HPCC Systems repository to
+          the helm repository list, add it now.</para>
+
+          <para><programlisting>helm repo add hpcc https://hpcc-systems.github.io/helm-chart/</programlisting></para>
+
+          <para>If you have added it, update to the latest charts:</para>
+
+          <para><programlisting>helm repo update</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Create a new text file and name it <emphasis
+          role="bold">threethorsonequeue.yaml</emphasis> and open it in a text
+          editor.</para>
+
+          <para>You can use any text editor.</para>
+        </listitem>
+
+        <listitem>
+          <para>Open the default values file that you saved earlier
+          (myvalues.yaml) in a text editor.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire <emphasis role="bold">thor:</emphasis> section
+          and paste it into the new threethorsonequeue.yaml file.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire contents of the new yaml file, except the
+          first line (thor:), and paste it at the end of the file <emphasis
+          role="bold">twice</emphasis>.</para>
+        </listitem>
+
+        <listitem>
+          <para>Edit the file in the following manner:</para>
+
+          <para><orderedlist>
+              <listitem>
+                <para>Give each Thor a unique value for <emphasis
+                role="bold">name:</emphasis>.</para>
+
+                <para>In this example, we use <emphasis
+                role="bold">thor400-az1, thor400-az2, </emphasis>and <emphasis
+                role="bold">thor400-az3</emphasis>.</para>
+              </listitem>
+
+              <listitem>
+                <para>Add an <emphasis role="bold">auxQueues:</emphasis> entry
+                to each Thor block using a common name</para>
+
+                <para>In this example, we are using </para>
+
+                <para><emphasis role="bold">auxQueues: [ thor400-azs
+                ]</emphasis></para>
+              </listitem>
+
+              <listitem>
+                <para>Make sure the <emphasis role="bold">prefix:</emphasis>
+                is the same in each Thor block.</para>
+              </listitem>
+            </orderedlist></para>
+        </listitem>
+
+        <listitem>
+          <para>Save the file and close the text editor.</para>
+
+          <para>The resulting threethorsonequeue.yaml file should look like
+          this</para>
+
+          <para><emphasis role="bold">Note:</emphasis> The comments have been
+          removed to simplify the example:</para>
+
+          <para><programlisting>thor:
+- name: thor400-az1
+  auxQueues: [ thor400-azs ]
+  maxGraphs: 1
+  maxJobs: 2
```

Comment:
It is best to make maxJobs == maxGraphs in these configuration.

Because, otherwise, each Thor that is listening to thor400-azs, is at liberty to pull off (in this case 2 job), but will only be able to run 1 on Thor (maxGraphs). So this configuration will confuse the onlooker who will see jobs go to a Thor that is already busy running a portion of its job in a Thor instance.

Perhaps the whole example should be scaled back too. e.g. numWorkersPerPod is an edge feature too, and not related to this feature.
I'd also for the purposes of the example stick to something more similar to the original yaml, e.g. numWorkers: 4 and naming like thor1, thor2 etc.